app/src/main/res/values/preference_keys.xml

```diff
@@ -189,7 +189,9 @@
 
     <!-- Wallpaper Settings -->
     <string name="pref_key_wallpapers" translatable="false">pref_key_wallpapers</string>
+    <string name="pref_key_previous_wallpaper" translatable="false">pref_key_previous_wallpaper</string>
```

Comment:
Let's talk about this offline.  We're trying to move away from adding shared preferences for telemetry.  Let's explore if there's another way we can avoid adding these.